src/schema/artist/current.js

```diff
@@ -10,25 +10,36 @@ const CurrentEventType = new GraphQLObjectType({
     image: {
       type: Image.type,
     },
-    headline: {
+    status: {
```

Comment:
Hmm... I'm not sure I completely follow.

Reply:
Basically, the fields from `CurrentEvent` are basically just sub fields from `Sale` or `Show`, so making this an interface or a union wouldn't be inappropriate. It might even be more idiomatic. This is compared to the other option of copying some of the resolvers (or extracting them into their own helpers) from `Sale` and `Show` into here (for instance, to be able to resolve `href` properly).

Reply:
I don't think it really matters at the end of the day though, so just sharing those resolvers here is totally fine.